.github/workflows/quality-checks.yml

```diff
@@ -16,23 +16,27 @@ jobs:
             - name: Checkout
               uses: actions/checkout@v2
 
+            - uses: actions/setup-node@v2
+              with:
+                node-version: '14'
+
             - name: Install Dependencies
-              run: yarn
+              run: npm i --ignore-scripts

```

Comment:
yarn uses its own node version so I have faced an issue where TS locally compiled successfully but failed in some steps on GH. Since we use npm during development I used it also here and set a proper node version so that we have consistent setup.